packages/search-ui/src/__tests__/DebounceManager.test.js

```diff
@@ -0,0 +1,111 @@
+import DebounceManager from "../DebounceManager";
+
+describe("#runWithDebounce", () => {
+  it("will run the provided function after the provided number of milliseconds", () => {
+    const manager = new DebounceManager();
+    let debounced = jest.fn();
+
+    manager.runWithDebounce(1000, "debounced", () => {
```

Comment:
Correct me if I'm wrong, but can we just do
`manager.runWithDebounce(1000, "debounced", debounced);`
instead of using an anonymous function?

Reply:
Same question below for all other instances of
```
() => {
      debounced();
    }
```

Reply:
Yes, good catch.

Reply:
_rubbing hands in anticipation of all the extra lines saved_ :trollface: